<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ampInpLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Amplitude</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ampInp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>20</number>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="anaInp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>20</number>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="anaInpLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Analog Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="phaseInpLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Phase Shift</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="phaseInp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>20</number>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="drawBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QLabel" name="sampInpLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sampling Frequency</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sampInp">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>20</number>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Wave Type</string>
    </property>
    <widget class="QRadioButton" name="isSinBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Is SIN Wave</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="isCosBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Is COS Wave</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuimport">
    <property name="title">
     <string>import</string>
    </property>
    <addaction name="txtDis"/>
    <addaction name="txtCon"/>
    <addaction name="txtDisCon"/>
   </widget>
   <addaction name="menuimport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="txtDis">
   <property name="text">
    <string>txt to discrete wave</string>
   </property>
  </action>
  <action name="txtCon">
   <property name="text">
    <string>txt to continous wave</string>
   </property>
  </action>
  <action name="txtDisCon">
   <property name="text">
    <string>txt to discrete and continous</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
